Common/src/main/java/gameLogic/ShipManager.java

```diff
@@ -0,0 +1,44 @@
+package gameLogic;
+
+import java.awt.*;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class ShipManager implements Serializable {
+
+    private Set<Ship> ships;
+
+    public void initShips(Set<Ship> ships) {
+        this.ships = ships;
+    }
+
+    Set<Point> getPointsAroundSunkShip(Point point) {
+        Set<Point> aroundSunkCells = new HashSet<>();
+        ships.stream()
+                .filter(ship -> ship.containsPoint(point))
+                .forEach(ship -> ship.getNeighbours()
+                        .stream()
+                        .forEach(aroundSunkCells::add));
+        return aroundSunkCells;
+    }
+
+    Set<Point> getPointsOfSunkShip(Point point) {
```

Comment:
getPointsAroundSunkShip & getPointsOfSunkShip - code duplication (just one method invocation is defferent)